src/main/java/com/mathworks/ci/MatlabBuild.java

```diff
@@ -70,7 +72,9 @@ default void copyFileInWorkspace(String sourceFile, String targetFile, FilePath
         targetFilePath.chmod(0755);
     }
 
-    default FilePath getFilePathForUniqueFolder(Launcher launcher, String uniqueName,FilePath workspace)
+    @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE",
+            justification = "workspace is passed through perform method, which will always have NonNull FilePath")

```

Comment:
Would it be hard to fix this issue rather than suppress it? Could you just do a null check? I tend to prefer suppressing issues only as a last resort.

Reply:
I thought of this as well , however my rational was if workspace for any reason is null then build would fail much before it reaches to perform() method of builder class. However i also tried wrapping it around few null checks like Objects.requireNonNull() and few other but it seems there is open issue for this on find bug https://github.com/spotbugs/spotbugs/issues/651 

Current fix with if(cmp ==null) only seem to be working so far.